closes #136 Start shift of color scheme

martensonbj opened the pull request:
Very much still working on this but initial attempt to transition color theme into something more consistant has begun. 

Feedback appreciated! Let me know if you hate colors.

martensonbj committed:
Start shift of color scheme

martensonbj committed:
Add home page title

martensonbj committed:
Added jquery for background carousel

martensonbj committed:
Changed img tag to rails helper tag

ToniRib commented:
Loooove the new front page title!

ToniRib commented:
Overall I like the new color scheme. For some reason in the cart it looks like the Update and Delete buttons aren't lined up anymore though. I made some minor edits to that and some of the smaller font sizes. Added the file upload option to the admin create item form and did a little styling there also.

ToniRib committed:
Resize small fonts, update styling on forms

Some of the fonts on the various pages were very small, so they are now
a little more readable. In addition, this adds some styling to the forms
for adding items (both admin and artist) and for updating the user's profile.

ToniRib commented:
@martensonbj Take a look at the changes and let me know what you think!

martensonbj committed:
Make home page colors darker

martensonbj committed:
Fix merge conflicts

martensonbj commented on config/initializers/assets.rb:
```diff
@@ -10,4 +10,4 @@
 # Rails.application.config.assets.precompile += %w( search.js )
 
-Rails.application.config.assets.precompile += %w( favicon.ico )
+Rails.application.config.assets.precompile += %w( favicon.ico bg1.jpg bg2.jpg bg4 )
```
I'm intentionally ignoring this.

martensonbj committed:
Merge branch 'master' of github.com:martensonbj/little_shop into 136_Change-color-theme

martensonbj commented:
I merged in the cycle background image branch into this branch so now I"m not sure what happens. But the background image should cycle through decently sized images, color scheme has been tweaked, and current master merged in. Let me know if anyone has any further comments!

martensonbj committed:
Merge branch 'master' of github.com:martensonbj/little_shop into 136_Change-color-theme

ToniRib committed:
Specify which button to click with the carousel
